portfolio/src/main/webapp/index.html

```diff
@@ -9,21 +9,21 @@
   </head>
   <body>
     <div id="content">
-        <div id="header">
-            <div id="left"><img src="/images/profile.jpg" id="profile"></div>
-            <div id="right">
-                <h1>Hi, I'm Keely!</h1>
-                <p>I'm a sophomore studying computer science at Cornell University (and exploring economics and 
-                    business). Outisde of school, I'm an avid hiker, recreational tennis player, and huge bookworm. 
-                    Currently re-reading <i>Gone with the Wind</i> - any book recommendations are always welcome! 
-                </p>
-                <p>Check out my <a href="https://www.linkedin.com/in/keely-wan/">LinkedIn</a> or my 
-                <a href="https://www.github.com/keelywan">GitHub</a> for some of my work!</p>
-            </div>
+      <div id="header">
+        <div id="left"><img src="/images/profile.jpg" id="profile"></div>
+        <div id="right">
+          <h1>Hi, I'm Keely!</h1>
+          <p>I'm a sophomore studying computer science at Cornell University (and exploring economics and 
+            business). Outisde of school, I'm an avid hiker, recreational tennis player, and huge bookworm. 
+            Currently re-reading <i>Gone with the Wind</i> - any book recommendations are always welcome! 
+          </p>
+          <p>Check out my <a href="https://www.linkedin.com/in/keely-wan/">LinkedIn</a> or my 
+            <a href="https://www.github.com/keelywan">GitHub</a> for some of my work!</p>
         </div>
-        <!-- <p>Click here to get a random message:</p>
-        <button onclick="addRandomGreeting()">Hello</button>
-        <div id="greeting-container"></div> -->
+      </div>
+      <!-- <p>Click here to get a random message:</p>
```

Comment:
I would delete this commented out code. It's better to keep changes like this that you are considering adding later in local branches that aren't out for review.